catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -74,6 +75,14 @@
 
     private final Executor configurationExecutor;
 
+    private static final int MAX_THREAD_SIZE = 8;
```

Comment:
This maximum was selected from the guidance in this ticket : https://codice.atlassian.net/browse/DDF-2756